x/auth/client/cli/query.go

```diff
@@ -100,6 +100,19 @@ func GetAccountCmd() *cobra.Command {
 			queryClient := types.NewQueryClient(clientCtx)
 			res, err := queryClient.Account(cmd.Context(), &types.QueryAccountRequest{Address: key.String()})
 			if err != nil {
+				node, err2 := clientCtx.GetNode()
```

Comment:
No need to use `err2`, you can safely shadow the existing `err`

Reply:
In a case, where we have an `err` from L101 and `err` returned as `nil` from L103 and L107 and `catchingUp` is `false`, then err would be nil which isn't true. I think having a separate variable makes sense here.